<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArithmeticFilterInterface</class>
 <widget class="QWidget" name="ArithmeticFilterInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>213</width>
    <height>281</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>213</width>
    <height>281</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>213</width>
    <height>281</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>4</y>
     <width>207</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="volumeAHLayout" stretch="1,2">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="volumeALabel">
      <property name="text">
       <string>Volume A:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="volumeAComboBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>29</y>
     <width>207</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="volumeBHLayout" stretch="1,2">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="volumeBLabel">
      <property name="text">
       <string>Volume B:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="volumeBComboBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>54</y>
     <width>207</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="volumeCHLayout" stretch="1,2">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="volumeCLabel">
      <property name="text">
       <string>Volume C:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="volumeCComboBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>244</y>
     <width>205</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="replaceInvertLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QCheckBox" name="replaceCheckBox">
      <property name="text">
       <string>Replace</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="invertButton">
      <property name="text">
       <string>Run Filter</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>79</y>
     <width>207</width>
     <height>50</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Example Expressions:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>128</y>
     <width>205</width>
     <height>117</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Expressions:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
